core/shared/src/main/scala/org/http4s/internal/Logger.scala

```diff
@@ -71,8 +71,8 @@ object Logger {
   )(log: String => F[Unit])(implicit F: Monad[F]): F[Unit] = {
     def prelude =
       message match {
-        case req: Request[_] => s"${req.httpVersion} ${req.method} ${req.uri}"
-        case resp: Response[_] => s"${resp.httpVersion} ${resp.status}"
+        case req: Request[F @unchecked] => s"${req.httpVersion} ${req.method} ${req.uri}"
+        case resp: Response[F @unchecked] => s"${resp.httpVersion} ${resp.status}"
```

Comment:
I'm not sure why making `Media` covariant caused scalac to start complaining about the `_`.